modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -176,16 +179,22 @@ const Modal = ({
 		<>
 			{visible && (
 				<ClayModal
+					aria-labelledby={modalLabelId}
+					center={center}
 					className="liferay-modal"
 					containerProps={{...containerProps}}
 					disableAutoClose={disableAutoClose}
 					id={id}
 					observer={observer}
+					role="dialog"
 					size={url && !size ? 'full-screen' : size}
 					status={status}
 					zIndex={zIndex}
 				>
-					<ClayModal.Header className={headerCssClass}>
+					<ClayModal.Header
+						className={headerCssClass}
+						id={modalLabelId}
```

Comment:
Using `aria-labelledby` and `id` inside Modal header was necessary to improve accessibility requested by aXe. I had two options:

1. `aria-labelledby` + `id` on heading element
2. use `title` attr  on modal

1:

<img width="636" alt="Screen Shot 2022-03-07 at 20 38 32" src="https://user-images.githubusercontent.com/7663513/157136629-279acdf7-9657-439e-84e7-671350d00931.png">

2:
<img width="643" alt="Screen Shot 2022-03-07 at 20 48 48" src="https://user-images.githubusercontent.com/7663513/157137637-12f35bfc-892d-4869-9100-79eb67f211ee.png">

I  decided go with 1) since it has a more clear message. 

@pat270 @marcoscv-work @edalgrin, what are your opinions on this?

Reply:
I'll leave this call for our style experts above :)

Just a warning for 
> use title attr on modal

This might have unintentional interaction with tooltips.

Reply:
Sorry I'm late! I also prefer the first solution. `title` is often a tricky attribute, `aria-labelledby` is (probably) less code and help to keep the title in sync (screen-readers and visuals)

Reply:
I also remember something like "using `<section>` and `<h*>` (h1, h2, ...) we might not need the `aria-labelledby`" but I'm not sure of it, so it would be interesting to test it or wait for someone else comment